### PR TITLE
statistics: process the case that we lack a column in stats table

### PR DESCRIPTION
**Pull request by hanfei1991**
When we can't find a column in stats table, we return a pseudo table for now. It's not reasonable.
We should return a real table and use pseudo estimation for the missed histogram.

@shenli @coocood @zimulala @lamxTyler PTAL

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

File: statistics/table.go [hunk 1]
```diff
@@ -132,8 +132,6 @@ func (h *Handle) tableStatsFromStorage(tableInfo *model.TableInfo, count int64)
 	}
 	// indexCount and columnCount record the number of indices and columns in table stats. If the number don't match with
```
Comment (lamxTyler) [inline statistics/table.go]: The comment can be removed?

**Comment by coocood** [comment]
LGTM

**Pull request merged**